src/main/scala/org/apache/mesos/chronos/scheduler/jobs/stats/JobStats.scala

```diff
@@ -360,7 +353,16 @@ class JobStats @Inject()(clusterBuilder: Option[Cluster.Builder], config: Cassan
     }
   }
 
-  def jobQueued(job: BaseJob, attempt: Int) {
+  override def onEvent(event: JobEvent): Unit = event match {
+    case JobRemoved(job) => removeJobState(job)
+    case JobQueued(job, taskId, attempt) => jobQueued(job, taskId, attempt)
+    case JobStarted(job, taskStatus, attempt) => jobStarted(job, taskStatus, attempt)
+    case JobFinished(job, taskStatus, attempt) => jobFinished(job, taskStatus, attempt)
+    case JobFailed(job, taskStatus, attempt) => jobFailed(job, taskStatus, attempt)
+    case _ => // do nothing
```

Comment:
I don't think so, JobStats is not (and never was) interested in the other events. If you want to add some kind of info or debug logging, that would be ok